internal/dataplane/kongstate/util.go

```diff
@@ -257,21 +258,14 @@ func SecretToConfiguration(
 	return config, nil
 }
 
-// PrettyPrintServiceList makes a clean printable list of a map of Kubernetes
+// PrettyPrintServiceList makes a clean printable list of Kubernetes
 // services for the purpose of logging (errors, info, e.t.c.).
-func PrettyPrintServiceList(services map[string]*corev1.Service) string {
-	var serviceList string
-	first := true
+func PrettyPrintServiceList(services []*corev1.Service) string {
```

Comment:
```suggestion
func prettyPrintServiceList(services []*corev1.Service) string {
```
nit: it is used in the same package hence it doesn't need to be exported

Reply:
This is only OK because we're releasing a major + it's in `internal/`. Otherwise it wouldn't be as OK :)